static/app/views/alerts/rules/details/body.tsx

```diff
@@ -251,6 +248,12 @@ export default class DetailsBody extends React.Component<Props> {
                 value={<TimeSince date={rule.dateModified} suffix={t('ago')} />}
               />
             )}
+            <KeyValueTableRow
+              keyName={t('Team')}
+              value={
+                teamActor ? <ActorAvatar actor={teamActor} size={24} /> : 'Unassigned'
```

Comment:
should this be 
```suggestion
                teamActor ? <ActorAvatar actor={teamActor} size={24} /> : t('Unassigned')
```
?